halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/services/v1/AccountService.java

```diff
@@ -42,7 +42,10 @@
   ValidateService validateService;
 
   public List<Account> getAllAccounts(NodeReference reference) {
-    NodeFilter filter = new NodeFilter(reference).withAnyHalconfigFile().withAnyAccount();
+    NodeFilter filter = NodeFilter.makeRejectAllFilter()
```

Comment:
I get it, but reads odd to say reject everything then extend that with other stuff as opposed to creating an empty filter and building it up. The implementation is exactly the same, but the name of that empty filter is what makes this read odd.

Why not makeEmptyFilter() instead of makeRejectAllFilter()?